File: Python/get_max.py
import numpy as np

def get_max(arr,threshold,num_pnts):
    out = []
    if len(arr) == 0:
        return np.array(out)
    for i in range(num_pnts):
        [a, b] = arr.max(0), arr.argmax(0)
        if(a<threshold):
            return np.array(out)
        else:
            out.append(b)
            arr[b] = 0
    return np.array(out)

File: Matlab/get_max.m
function [out] = get_max(arr,threshold,num_pnts)
%GET_MAX, this function returns uptil num_pnts many maximums above
% 'threshold'

    out = [];
    for i = 1:num_pnts
        [a b] = max(arr);
        if(a < threshold)
            return;
        else
            out = [out b];
            arr(b) = 0;
        end
    end

end

